docs/general/builtins/scalar.rst

```diff
@@ -2402,6 +2402,33 @@ Example:
     +--------------+
     SELECT 1 row in set (... sec)
 
+.. _version:
+
+``version``
```

Comment:
Should we point out that this function is registered under the `pg_catalog` schema?

Reply:
Maybe yes, although we don't describe this for the other system information functions like e.g. `pg_typeof`, `current_schema` etc.
Shall we may note this there as well?

Reply:
Not sure. Also fine for me to omit it.